core/node/libp2p/transport.go

```diff
@@ -21,7 +27,7 @@ func Transports(pnet struct {
 	return opts
 }
 
-func Security(enabled bool) interface{} {
+func Security(enabled bool, securityTransportOverride []string) interface{} {
```

Comment:
@Stebalien Why have we added the `securityTransportOverride ` option ? Is there any use case for it right now ?

Reply:
At the moment, go-ipfs prefers TLS, then SECIO, then Noise. That means the noise transport will _never_ be used by default. The override option lets us actually _use_ noise by:

* Putting it before TLS.
* Removing SECIO.
* Etc...

See the tests and the new experimental-features documentation for details. If it doesn't make sense, I'll expand the documentation in a followup PR.